src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -755,7 +775,33 @@ pub const RTLD_DI_TLS_MODID: c_int = 9;
 pub const RTLD_DI_TLS_DATA: c_int = 10;
 
 pub const SOCK_NONBLOCK: c_int = O_NONBLOCK;
+
 pub const PIDFD_NONBLOCK: c_uint = O_NONBLOCK as c_uint;
+pub const PIDFD_THREAD: c_uint = O_EXCL as c_uint;
+
+pub const PIDFD_SIGNAL_THREAD: c_uint = 1;
+pub const PIDFD_SIGNAL_THREAD_GROUP: c_uint = 2;
+pub const PIDFD_SIGNAL_PROCESS_GROUP: c_uint = 4;
+
+pub const PIDFD_INFO_PID: c_uint = 1;
+pub const PIDFD_INFO_CREDS: c_uint = 2;
+pub const PIDFD_INFO_CGROUPID: c_uint = 4;
+pub const PIDFD_INFO_EXIT: c_uint = 8;
```

Comment:
To keep these more similar to the original sources, could you use `1 << n`  for the values?